doc/rules/operator/no_useless_concat_operator.rst

```diff
@@ -0,0 +1,58 @@
+===================================
+Rule ``no_useless_concat_operator``
+===================================
+
+There should not be useless concat operations.
+
+Configuration
+-------------
+
+``juggle_simple_strings``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Allow for simple string quote juggling if it results in more concat-operations
+merges.
+
+Allowed types: ``bool``
+
+Default value: ``false``
+
+Examples
+--------
+
+Example #1
+~~~~~~~~~~
+
+*Default* configuration.
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+    <?php
+   -$a = 'a'.'b';
+   +$a = 'ab';
+
+Example #2
+~~~~~~~~~~
+
+With configuration: ``['juggle_simple_strings' => true]``.
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+    <?php
+   -$a = 'a'."b";
+   +$a = "ab";
```

Comment:
I don't understand the `juggle_simple_strings` option because I don't know what the default behavior would have done on this.

Maybe more concatenations need to be added to example 1 and 2.
The purpose would be to have the same input, but different output.

Reply:
Also, example with line breaks need to be added

Reply:
I tried to explain that this cannot work, the doc's are generated based on the example and show the `diff`, you asking for examples where things don't change, if there is no change than there is no diff and it won't show in the docs.